packages/swirl-components/src/components/swirl-color-input/swirl-color-input.stories.ts

```diff
@@ -16,8 +16,7 @@ const Template = (args) => {
   const formControl = document.createElement("swirl-form-control");
   const element = generateStoryElement("swirl-color-input", args);
 
-  formControl.setAttribute("label", "Color input");
-  formControl.setAttribute("disabled", args.disabled ? "true" : "false");
+  addArgumentsToElement(formControl, { ...args, label: "Color input" });
```

Comment:
I was a bit undecided about the approach to take here...

Initially I thought about creating a helper function in which I specifically added the attributes (description, disabled, inline, invalid, label ), but then I thought that if in the future we add any more attributes to the form-control, we would have to remember to update the function.

So I thought of this solution of injecting all the attributes of the input into the form-control. The disadvantage of this solution is that we'll be adding attributes we don't need to the form-control.
In your opinion, does that feel unsafe? Should I go with the more declarative approach?

Reply:
`args` only contains custom props that should always be distinct from native HTML attributes. So adding them to components that might not support them should not be an issue.